### ISSUE TITLE
linux does not recognize the vacuum cleaner

### ISSUE THREAD

**Issue by razserv2010** [rumpeltux/viomi-rooting#34]
My vacuum cleaner is: STYTJ02YM (viomi-v8)
I followed the instructions until I got to the point where I need to connect the vacuum cleaner to the computer via USB
I plugged it in and after a minute the vacuum plays "turn on" and nothing happens.
* I have tried more than 20 times to disconnect and reconnect
* I tried several USB DATA cables
* And I also tried to check an Android phone to check if the computer recognizes the phone as adb, it does recognize the phone as adb
Do I have to press anything on the vacuum cleaner for it to enter adb mode?

**Comment by benoegen** [comment]
#31 seems to be the same issue as this

**Comment by razserv2010** [comment]
> #31 seems to be the same issue as this

Anyone found a solution?

**Comment by Dropaq** [comment]
@razserv2010 yes! https://github.com/rumpeltux/python-miio/issues/1#issuecomment-915647117
